gateway/api/server.go

```diff
@@ -447,4 +448,20 @@ func (api *Api) buildRoutes(route *gin.RouterGroup) {
 		api.AllowApiKey,
 		api.Authenticate,
 		apiserverinfo.New(api.GrpcURL).Get)
+
+	// Jira Integration routes
+	route.GET("/integration/jira",
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		apijiraintegration.Get)
+
+	route.POST("/integration/jira",
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		apijiraintegration.Post)
+
+	route.PUT("/integration/jira",
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		apijiraintegration.Put)
```

Comment:
following rest api best practices, the entity should be in plural form, being `integrations/jira`